df_to_azure/__init__.py

```diff
@@ -2,7 +2,7 @@
 
 from .export import df_to_azure
 
-__version__ = "0.7.0-rc.1"
+__version__ = "0.8.0-rc.1"
```

Comment:
Eventueel release `0.7.0-rc.2` van maken? Aangezien `0.7.0` nooit gereleased is.

Reply:
Yes, I also still saw an open PR related to version number so was not sure what to do with this.